event_registration_cancel_reason/wizard/event_registration_cancel_log_reason.py

```diff
@@ -21,9 +21,16 @@ class EventRegistrationCancelLogReason(models.TransientModel):
     def default_get(self, var_fields):
         res = super(EventRegistrationCancelLogReason, self).default_get(
             var_fields)
-        registration = self.env['event.registration'].browse(
-            self.env.context['active_id'])
-        res['event_type_id'] = registration.event_id.type.id
+        registrations = self.env['event.registration'].browse(
+            self.env.context['active_ids'])
+        event_type = registrations.mapped("event_id.type")
+        try:
+            event_type.ensure_one()
+        except exceptions.except_orm:
```

Comment:
Python's common style is [EAFP](https://docs.python.org/2/glossary.html#term-eafp). Anyway change it if you don't like it, it's not a blocker.